src/icons/fileIcons.ts

```diff
@@ -858,5 +858,8 @@ export const fileIcons: FileIcons = {
         { name: 'disc', fileExtensions: ['iso'] },
         { name: 'fortran', fileExtensions: ['f', 'f77', 'f90', 'f95', 'f03', 'f08'] },
         { name: 'liquid', fileExtensions: ['liquid'] },
+        { name: 'shader-vert', fileExtensions: ['vert', 'vertexshader'] },
```

Comment:
Are these enough? As said in the issue suggestions, there are a few more that might be used.

Reply:
You are right, shaders do not have a uniform extension. Some projects will use .vs/.fs as the file extension, but .fs is also used by fsharp.
> { name: 'fsharp', fileExtensions: ['fs', 'fsx', 'fsi', 'fsproj'] },

Reply:
Hm thats a good point, nevermind then 😅

Reply:
but there are some "standard" extensions, you could also add .glsl, .frag